test/lru.ml

```diff
@@ -52,16 +66,24 @@ struct
   let discard () =
     let cap = 10 in
     let t = Lru.v cap in
+
+    let d = Hashtbl.create cap in
+
     let all_values = add_fresh_values ~check:false t (2 * cap) in
     let still_present, removed = split all_values cap in
     List.iter
       (fun k ->
         Alcotest.(check bool)
-          "Recently added values are still present" true (Lru.mem t k))
+          "Recently added values are still present" true (Lru.Cache.mem t k))
       still_present;
     List.iter
       (fun k ->
-        Alcotest.(check bool) "Old values have been removed" false (Lru.mem t k))
+        Alcotest.(check bool)
+          "Old values have been removed" false (Lru.Cache.mem t k))
+      removed;
+    List.iter
+      (fun k ->
+        Alcotest.(check bool) "Old values have been removed" false (DB.mem d k))
```

Comment:
I'm not sure I get this. `d` is created, but nothing is ever added inside, so what does this test exactly?

Reply:
hummmm, I think just to test if all values still be present in the DB. (maybe not a test relly interessant)